client/my-sites/domains/domain-management/settings/cards/contact-information/contacts-card.tsx

```diff
@@ -0,0 +1,188 @@
+import { Button, Card } from '@automattic/components';
+import { ToggleControl } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { PRIVACY_PROTECTION, PUBLIC_VS_PRIVATE } from 'calypso/lib/url/support';
+import ContactDisplay from 'calypso/my-sites/domains/domain-management/contacts-privacy/contact-display';
+import {
+	domainManagementEditContactInfo,
+	domainManagementManageConsent,
+} from 'calypso/my-sites/domains/paths';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import {
+	disableDomainPrivacy,
+	discloseDomainContactInfo,
+	enableDomainPrivacy,
+	redactDomainContactInfo,
+} from 'calypso/state/sites/domains/actions';
+import { isUpdatingDomainPrivacy } from 'calypso/state/sites/domains/selectors';
+import type { ContactsCardPassedProps, ContactsCardProps } from './types';
+
+const ContactsPrivacyCard = ( props: ContactsCardProps ): JSX.Element => {
+	const translate = useTranslate();
+	const togglePrivacy = () => {
+		const { selectedSite, privateDomain, selectedDomainName: name } = props;
+
+		if ( privateDomain ) {
+			props.disableDomainPrivacy( selectedSite.ID, name );
+		} else {
+			props.enableDomainPrivacy( selectedSite.ID, name );
+		}
+	};
+
+	const toggleContactInfo = () => {
+		const { selectedSite, contactInfoDisclosed, selectedDomainName: name } = props;
+
+		if ( contactInfoDisclosed ) {
+			props.redactDomainContactInfo( selectedSite.ID, name );
+		} else {
+			props.discloseDomainContactInfo( selectedSite.ID, name );
+		}
+	};
+
+	const getPrivacyProtection = () => {
+		const { privateDomain, privacyAvailable } = props;
+		const { isUpdatingPrivacy } = props;
+
+		const label = privateDomain
+			? translate( 'Privacy protection on' )
+			: translate( 'Privacy protection off' );
+
+		let privacyProtectionNote;
+		if ( ! privacyAvailable ) {
+			privacyProtectionNote = (
+				<p className="contact-information__toggle-item">
+					{ translate(
+						'Privacy protection is not available due to the registry’s policies. {{a}}Learn more{{/a}}',
+						{
+							components: {
+								a: <a href={ PRIVACY_PROTECTION } />,
```

Comment:
I noticed that previous component had` target="blank"` for this link, maybe we can maintain the same approach here.

```suggestion
								a: <a href={ PRIVACY_PROTECTION } target="_blank" rel="noopener noreferrer" />,
```